### DISCUSSION TITLE
Welcome to stan_stans Discussions!

### DISCUSSION THREAD

**Discussion by ResearchActuary** [casact/stan_stans#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rajesh06** [comment]
Thanks Brian - I am a bit behind on the video lectures but I am hoping the catch up over the holiday break. 
Now to your request:
This is Rajesh (Raj) Sahasrabuddhe (my personal github account).
I am on the MAS IIC Exam Committee and McElreath is our primary text there.
We also fit Bayesian models at work semi-regularly. I think we already do a pretty good job (thanks to my colleague and fellow course member Kenny Smart).

I am hoping to "fine-tune" my Bayesian knowledge through this course.
